platform/android/app/src/main/res/values/strings.xml

```diff
@@ -32,9 +34,15 @@
     <string name="button_url_open_confirm">Yes</string>
     <string name="button_url_open_reject">No</string>
     <string name="button_update_notification_close">OK</string>
+
     <string name="auth_prompt_title">Login into your polyPod</string>
     <string name="auth_prompt_subtitle">Use device lock screen</string>
     <string name="auth_prompt_success">Authentication successful</string>
+
+    <string name="re_auth_prompt_title">Please confirm your Login.</string>
+    <string name="re_auth_prompt_subtitle">For security reasons, we ask you to re-authenticate before changing this setting</string>
+    <string name="re_auth_prompt_success">Re-authorized authentication successfully</string>
```

Comment:
I would like to propose to the following texts:

- re_auth_prompt_title = "Authenticate to continue"
- re_auth_prompt_subtitle = "Please authenticate using your device's screen lock"
- re_auth_prompt_success = "Authentication successful"
- auth_prompt_title = "Log in to your polyPod"
- auth_prompt_subtitle = "Please authenticate using your device's screen lock"

These re_auth_* texts should **only** be used for the case where authentication is already enabled and the user is trying to disable the setting. For the case where it's currently disabled and the user is trying to enable it the normal auth_prompt_* texts should be used. The re_auth_* texts are suitable for any kind of "step up authentication" scenarios we may need in the future. 

And the German texts...
- re_auth_prompt_title = "Authentifizieren um fortzufahren"
- re_auth_prompt_subtitle = "Bitte authentifizieren Sie sich mit dem Sperrbildschirm Ihres Gerätes"
- re_auth_prompt_success = "Authentifizierung erfolgreich"
- auth_prompt_title = "Einloggen in Ihren polyPod"
- auth_prompt_subtitle = "Bitte authentifizieren Sie sich mit dem Sperrbildschirm Ihres Gerätes"

Reply:
okay - this changes a lot the implementation. Let me have a look please